### PR TITLE
yabridge, yabridgectl: hard code wine path

### PR DESCRIPTION
**Pull request by kira-bruneau**
###### Motivation for this change
Adds `wineWowPackages.staging` as an explicit runtime dependency of `yabridge` so wine doesn't need to be installed separately.

###### Things done
- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by MetaDark** [comment]
Result of `nixpkgs-review pr 120627` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>yabridge</li>
    <li>yabridgectl</li>
  </ul>
</details>

**Comment by MetaDark** [comment]
I split the `WINELOADER` version fixes into a separate file and submitted a pull request upstream: https://github.com/robbert-vdh/yabridge/pull/95.

**Comment by robbert-vdh** [comment]
Just a heads up, there was a type error in the patch that I've fixed in https://github.com/robbert-vdh/yabridge/commit/93df3fa1da6ffcc69a5b384ba04e3da7c5ef23ef. You'll probably have to update your patches here to match!

**Comment by MetaDark** [comment]
Thanks @robbert-vdh! There weren't any problems with the Nix build, but I updated this commit to use the upstream patches.

**Comment by MetaDark** [comment]
Result of `nixpkgs-review pr 120627` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>yabridge</li>
    <li>yabridgectl</li>
  </ul>
</details>

File: pkgs/tools/audio/yabridgectl/default.nix [hunk 1]
```diff
@@ -17,6 +22,13 @@ rustPlatform.buildRustPackage rec {
 
   patchFlags = [ "-p3" ];
 
+  nativeBuildInputs = [ makeWrapper ];
+
+  postFixup = ''
+    wrapProgram "$out/bin/yabridgectl" \
+      --prefix PATH : "${wine}/bin"
```
Comment (SuperSandro2000) [inline pkgs/tools/audio/yabridgectl/default.nix:29]:
```suggestion
      --prefix PATH : ${lib.makeBinPath [ wine ]}
```

**Pull request merged**